.golangci.yaml

```diff
@@ -0,0 +1,17 @@
+# please keep this alphabetized
+linters:
+  enable: []
+  disable:
+    - asciicheck
+    - contextcheck
+    - forcetypeassert
+    - gocritic
+    - godot
+    - gofmt
+    - goimports
+    - misspell
+    - stylecheck
+
+run:
+  tests: true
+  timeout: 1m
```

Comment:
is 1 min enough? maybe 5 or 10, github action instances are slow some times

Reply:
oh I see everything is disabled, then 1 min is ok for now

Reply:
Note. There are still some linters that are enabled by default: `errcheck` and `gosimple` for example.

These are the ones that trigger the other fixes required in this changset.

I have bumped the timeout to 10m though.

Reply:
DO you think, the lint check should be an extra github action?